src/Workspaces/Core/Portable/FindSymbols/FindReferences/DependentTypeFinder_Remote.cs

```diff
@@ -29,7 +29,7 @@ internal static partial class DependentTypeFinder
                 var client = await RemoteHostClient.TryGetClientAsync(solution.Workspace, cancellationToken).ConfigureAwait(false);
                 if (client != null)
                 {
-                    var projectIds = projects?.SelectAsArray(p => p.Id) ?? default;
+                    var projectIds = projects?.Where(p => RemoteSupportedLanguages.IsSupported(p.Language)).SelectAsArray(p => p.Id) ?? default;
```

Comment:
Is this required for the fix, or just an optimization?

Reply:
This is required. Oop doesn't support other languages and crashes trying to find these projects on the other side.